### PR TITLE
#49 New Journal Button allows user to title new journals

### LINKED ISSUE #49: [FULLSTACK] Personalize Creation of New Journals
As a user I want to title my journal so that I can personalize them

Specs:
- Clicking the "New Journal" button should display a text field that allows user to input the name of the journal
- Clicking Enter or outside of the button should save the journal and display a new journal at the bottom of the sidebar
- Once the journal has been created, it should be saved into our database or localStorage
- Style should follow the root element

image

image

image

### PR DESCRIPTION
**Pull request by sjting8**
clicking the new journal button creates an untitled journal that can be renamed and after clicking enter it makes a call to the createJournal function

### CODE REVIEW

**Comment by camman00** [comment]
@reisandylamdjani you mind if I pass this to you? I am crashing right now lol

**Comment by reisandylamdjani** [comment]
yeahh I'll look at it

**Comment by p3reddy** [comment]
@camman00 @reisandylamdjani can you check this out and let me know what needs changing?

File: application/main.js
Comment (reisandylamdjani) [inline application/main.js:23]: is there a reason why we're commenting this out?

File: application/web/scripts/homepage/homepage.js
Comment (reisandylamdjani) [inline application/web/scripts/homepage/homepage.js]: Is there a reason why you're removing this? Cuz I think your functionality and the editing functionality is different?

**Review by reisandylamdjani — CHANGES_REQUESTED** [review]

File: application/main.js
Comment (p3reddy) [reply application/main.js:23]: It was originally commented out if you look at the file in the main branch, I just uncommented it for development purposes

File: application/web/scripts/homepage/homepage.js
Comment (p3reddy) [reply application/web/scripts/homepage/homepage.js]: The function didn't work and I think it should be done in a separate branch. Also having the function in another function seemed like bad practice. Also, the way it was written it would create a new article element every time which I changed to just rewriting the innerText of a single article element.

Comment (reisandylamdjani) [reply application/web/scripts/homepage/homepage.js]: I think it was merged to main for this, but it seems like Ben found a better solution so I guess it's okay

**Comment by reisandylamdjani** [comment]
Just wait for Cameron's approval but it look's good to me

**Comment by camman00** [comment]
@reisandylamdjani looks good I am going to approve but there were some existing merge conflicts I resolved. @sjting8 @azpi-arte can you please verify when you have the chance that this functionality works as expected?

**Pull request merged**
